tests/run_pretrained_models.py

```diff
@@ -42,12 +42,14 @@ def get_beach(shape):
 
 def get_random(shape):
     """Get random input."""
-    return np.random.sample(shape).astype(np.float32)
+    # add 1 to make sure generated data is not zero, for some ops will have a problem when encounter zero value, such as divide
+    return (np.random.sample(shape) + 1).astype(np.float32)
```

Comment:
for cases where data cannot be zero, they should handle at that time. This is generic random function, I don't think we should change it. 

another way to avoid 0, is to use numpy api directly in *.yaml to generate non-zero data.

Reply:
If we need this a lot you could also add get_random_none_zero()